readthedocs_build/config/config.py

```diff
@@ -14,7 +14,7 @@
     'load', 'BuildConfig', 'ConfigError', 'InvalidConfig', 'ProjectConfig')
 
 
-CONFIG_FILENAMES = ('readthedocs.yml', '.readthedocs.yml')
+CONFIG_FILENAMES = ('readthedocs.yml', '.readthedocs.yml', 'readthedocs.yaml')
```

Comment:
We need a `.readthedocs.yaml` too. Maybe we can refactor this to be a regex?

Reply:
Good point, I was considering even an helper method as this is not the only yaml file in use.

Reply:
Done, notice that now also `.readthedocs.yaml` is allowed. If you are happy with this implementation I'll document it but I'm not sure that listing 4 exact filenames with a slight difference is very human friendly, so I was thinking something along the lines of
... `readthedocs.yml` (optionally with `.` prefix, or `yaml` extension)

Most importantly: functionally it's now not possible to specify which file to prefer. There was a test that checked that files were found in the correct order (e.g. the order specified in the list), I had to reduce it to just testing whether it could find the files.